wasm-rpc-stubgen/src/lib.rs

```diff
@@ -43,22 +47,23 @@ test_r::enable!();
 #[derive(Parser, Debug)]
 #[command(name = "wasm-rpc-stubgen", version)]
 pub enum Command {
-    /// Generate a Rust RPC stub crate for a WASM component
+    /// [DEPRECATED] Generate a Rust RPC stub crate for a WASM component
```

Comment:
I think you can also mark these cases with `#[deprecated]` although never tried

Reply:
i don't want to make clippy sad